### PR TITLE
Handled delegate setup for cards

### PR DESCRIPTION
**Pull request by RakshithaAcharya**

### CODE REVIEW

File: MoEngagePluginBase/MoEngageCardsPluginManager.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//  MoEngageCardsPluginHandler.swift
+//  MoEngagePlugin
+//
+//  Created by Rakshitha on 16/08/23.
+//  Copyright © 2022 MoEngage. All rights reserved.
+//
+
+import Foundation
+
+class MoEngageCardsPluginManager: NSObject {
```
Comment (msoumya-engg-sdk) [inline MoEngagePluginBase/MoEngageCardsPluginManager.swift:11]: Can we generalize this for any plugin? Ideally Core shouldn't be aware of dependant modules and we shouldn't have to release core when onboarding new modules.

File: MoEngagePluginBase/MoEngageCardsPluginDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+//  MoEngageCardsPluginDelegate.swift
+//  MoEngagePlugin
+//
+//  Created by Rakshitha on 16/08/23.
+//  Copyright © 2022 MoEngage. All rights reserved.
+//
+
+import Foundation
+
+open class MoEngageCardsPluginDelegate {
```
Comment (msoumya-engg-sdk) [inline MoEngagePluginBase/MoEngageCardsPluginDelegate.swift:11]: Can we use protocols instead of class?

**Review by msoumya-engg-sdk — CHANGES_REQUESTED** [review]

File: MoEngagePluginCards/Helpers/MoEngagePluginCardsDelegateHandler.swift [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+//
+//  Created by Rakshitha on 16/08/23.
+//
+
+import Foundation
+import MoEngagePluginBase
+
+class MoEngagePluginCardsDelegateHandler: MoEngageCardsPluginDelegate {
+    override func initializeCardsPlugin() {
+       MoEngagePluginCardsBridge.sharedInstance
```
[the remark this answers is not in the document]
Comment (RakshithaAcharya) [reply MoEngagePluginCards/Helpers/MoEngagePluginCardsDelegateHandler.swift]: @msoumya-engg-sdk done

File: MoEngagePluginBase/MoEngageCardsPluginManager.swift (hunk 1 above)
Comment (RakshithaAcharya) [reply MoEngagePluginBase/MoEngageCardsPluginManager.swift]: @msoumya-engg-sdk done

File: MoEngagePluginBase/MoEngageCardsPluginDelegate.swift (hunk 2 above)
Comment (RakshithaAcharya) [reply MoEngagePluginBase/MoEngageCardsPluginDelegate.swift]: @msoumya-engg-sdk done

**Review by msoumya-engg-sdk — APPROVED** [review]
LGTM

**Pull request merged**